qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PatientEncounterSteps.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.EncounterFormPage;
+import org.openmrs.contrib.qaframework.page.ManageEncountersPage;
+import org.openmrs.contrib.qaframework.page.SystemAdministrationPage;
+
+public class PatientEncounterSteps extends Steps {
+
+	private AdministrationPage administrationPage;
+	private EncounterFormPage encounterFormPage;
+	private ManageEncountersPage manageEncountersPage;
+	private SystemAdministrationPage systemAdministrationPage;
+	private TestData.PatientInfo testPatient;
+	private static final String ENCOUNTER_DATE_TIME = "24/02/2022 00:00";
+	private static final String ROLE = "Unknown";
```

Comment:
can we rather use a known role?

Reply:
@kdaud  I have fixed according to the review requirements